Admin/AbstractAdmin.php

```diff
@@ -2017,7 +2017,7 @@ public function buildBreadcrumbs($action, MenuItemInterface $menu = null)
             E_USER_DEPRECATED
         );
         if ($this->breadcrumbsBuilder === null) {
-            $this->breadcrumbsBuilder = new BreadcrumbsBuilder();
+            $this->breadcrumbsBuilder = new BreadcrumbsBuilder($this->getConfigurationPool()->getContainer()->getParameter('sonata.admin.configuration.breadcrumbs'));
```

Comment:
> Granted, this piece of code is ugly, but will disappear with 4.0, and it's good to have all the ugly code in one place rather than also add a getter in the admin for this parameter, then remove it in 4.0, don't you think?

a PR afterwards which remove this stuff would be great, then